.github/workflows/ci.yml

```diff
@@ -997,6 +1004,10 @@ jobs:
         with:
           name: linux-launchers
           path: artifacts/
+      - uses: actions/download-artifact@v3
+        with:
+          name: linux-aarch64-launchers
+          path: artifacts/
```

Comment:
This was missing in the `update-packages` job. I hope this works now and the sdk -man script has all files available to upload them.

Reply:
Good catch. Yeah, it should work.